### ISSUE TITLE
go_test:runtime unit test sporadically runs out of memory on OS X

### ISSUE THREAD

**Issue by sodre** [conda-forge/go-feedstock#76]
This is the error message, and a [link][link] to the build log:

```
--- FAIL: TestGcSys (0.06s)
    gc_test.go:33: expected "OK\n", but got "using too much memory: 70813696 bytes\n"
FAIL
FAIL	runtime	9.366s
```

[link]: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=143315&view=logs&j=9c5ef928-2cd6-52e5-dbe6-9d173a7d951b&t=113b264d-9551-5065-da10-7eb3bdeaf1bc&l=1166

**Comment by sodre** [comment]
A workaround is to run the that test separately and see if the error goes away.

**Comment by sodre** [comment]
This is fixed in #80
